scripts/update_models.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
+
+set -e
+
+usage()
+{
+  echo "Usage: update_models.sh -c [ -v MODELS_VERSION_NUMBER ]"
+  exit 2
+}
+
+get_aws_sdk_go()
+{
+  # clone aws-sdk-go into folder
```

Comment:
Small nit, any chance you could do something like

```
DESTINATION_FOLDER="$1"
BRANCH_NAME="$2"
```

here and elsewhere to make it clear what the positional args are for? 🙏